configure.ac

```diff
@@ -1,20 +1,20 @@
 #                                               -*- Autoconf -*-/
 # Process this file with autoconf to produce a configure script.
 
-AC_PREREQ(2.59)
+AC_PREREQ([2.71])
```

Comment:
This seems too new, and not available yet in the latest Ubuntu (21.10).

Opened a PR to downgrade this requirement:
- https://github.com/virtualsquare/vde-2/pull/36